spec.md

```diff
@@ -260,6 +261,25 @@ circuit Foo :
 Functionality enabled by a layer is put in one or more layer blocks inside modules.
 For more information on layer blocks see [@sec:layer-blocks].
 
+Probe types may be colored with a layer to condition their existence on that layer being enabled.
+For more information on layer-colored probes see [@sec:probe-types] and [@sec:layer-coloring].
```

Comment:
The phrase "to condition their existence" sounds a bit existential. The very fabric of reality urges a slight rewording :) 

Perhaps: "Optionally, probe types may be colored with a layer. Layer coloring will indicate which layers must be enabled for a probe to be used."

Alternatively, if we want to resort to stronger language, we can do that too, but we would want to establish a deeper formalism for what that means. That is, we would need a technical term to describe when a probe is "not active" (or "disabled" or "erased" or whatever), and then we can cross-reference that term in the ABI to make that precise in terms of Verilog: that the probe does not exist when it's "not active".

Reply:
It's only problematic if the probes question their existence...

I switched to use a variant of your terse language in a fixup commit: https://github.com/chipsalliance/firrtl-spec/pull/160/commits/774cc117784c283c059426bf704c532a86de0f50

I like the simple "enabled"/"disabled" language. I'd almost prefer something even shorter ("on"/"off"), but we can bikeshed this in a cleanup over the language in the spec.